Microservices/event-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+spring.application.name=${SPRING_APPLICATION_NAME:event-service}
+server.servlet.context-path=${SERVER_SERVLET_CONTEXT_PATH:/api/event-service}
+logging.level.app.sportahub=${LOGGING_LEVEL_APP_SPORTAHUB:info}
+logging.level.org.springframework.web=${LOGGING_LEVEL_ORG_SPRINGFRAMEWORK_WEB:debug}
+
+# Swagger
+springdoc.api-docs.path=/api-docs
+springdoc.swagger-ui.path=/swagger-ui.html
+springdoc.swagger-ui.urls[0].name=Remote
+springdoc.swagger-ui.urls[0].url=https://api-dev.sportahub.app/api/event-service/api-docs
+springdoc.swagger-ui.urls[1].name=Local
+springdoc.swagger-ui.urls[1].url=http://localhost:8080/api/event-service/api-docs
+
+# Spring Data MongoDB
+spring.data.mongodb.uri=${MONGODB_URI}
```

Comment:
Can you replace this line by this: 
```yaml
spring.data.mongodb.uri=${MONGODB_URI:mongodb://root:password@sporta-host:27017/event-service?authSource=admin}
```

The reason for this is that your unit tests are trying to load this value for mongoConfig Bean and since there is no value in the `mongo.uri` when you run the test, the contextLoads() test fails.

Reply:
Got it, Done